### ISSUE TITLE
Question: Page with open text?

### ISSUE THREAD

**Issue by kenanhodzic** [blasten/turn.js#623]
How come author's [demo](http://www.turnjs.com/#samples/steve-jobs/6) have selectable text? Is it **html** or **PDF**? How to get that feature? Thanks.

**Comment by wsygdiv** [comment]
I have the same question.  is it html?  @iwege  @idlesysop  @blasten  @kenanhodzic .  how to use?  thanks.

**Comment by idlesysop** [comment]
@wsygdiv @kenanhodzic that is correct, it is html.   If you look in the demos folder, you can see examples of how to use it.  Keep in mind that this repo hasn't been actively maintained.  So, you may have some issues with the latest version of dependencies.

**Comment by sonicaaaaaa** [comment]
If you're considering HTML on the pages of your flipbook you might want to take a look at https://bubblin.io

**Comment by wsygdiv** [comment]
Sorry, I may not have made it clear.  I mean the content of  the pages can be html ?  the demos show content is Image format .  Can the demo use the html in pages , not image . Thanks ! @idlesysop

**Comment by tridnguyen** [comment]
@wsygdiv, yes, content can be html.

**Comment by wsygdiv** [comment]
@tridnguyen  do you have the demo for  html fomat ?  I don't know how to use it from Existing demo . thanks very much .

**Comment by tridnguyen** [comment]
@wsygdiv the demos in the demo folder have examples for you.  Each div inside the div with the "magazine" ID is a page.  You just need to place your html inside those child divs:
```
<div id="magazine">
  <div>Your html goes here for page one</div>
  <div>Your html goes here for page two</div>
</div>
```

**Comment by wsygdiv** [comment]
@tridnguyen thank you for your anwser . then,  can it perform automatic paging ? if I upload full book of html fomat , how to paging the book ?
